tests/conftest.py

```diff
@@ -201,3 +204,44 @@ def compare(received_raw: bytes, golden_name: str):
         assert diff_pixels == 0
 
     return compare
+
+
+class RemoteServer:
+    def __init__(self, browser_name: str) -> None:
+        driver_dir = Path(inspect.getfile(playwright)).parent / "driver"
+        if sys.platform == "win32":
+            node_executable = driver_dir / "node.exe"
+        else:
+            node_executable = driver_dir / "node"
+        cli_js = driver_dir / "package" / "lib" / "cli" / "cli.js"
+        self.process = subprocess.Popen(  # type: ignore
+            [node_executable, cli_js, "launch-server", browser_name],
+            stdout=subprocess.PIPE,
+            stderr=sys.stderr,
+            cwd=driver_dir,
+        )
+        assert self.process.stdout
+        self.ws_endpoint = self.process.stdout.readline().decode().strip()
+
+    def kill(self):
+        # Send the signal to all the process groups
+        if sys.platform == "win32":
+            subprocess(f"taskkill /F /PID {self.process.pid}")
```

Comment:
Missing `check_call` 😆

Reply:
Please do not comment everything, I will see it when its red on the bots :)